eureka_ml_insights/metrics/metrics_base.py

```diff
@@ -144,13 +144,39 @@ def __evaluate__(self, answer_text, target_text, is_valid):
         else:
             return "incorrect"
 
+class ExactOrMatch(ExactMatch):
```

Comment:
Could you please give a more descriptive name to this class and its child class? Maybe MultiCandidateAnyExactMatch

Reply:
yes

Reply:
Removed MultiCandidateAnyExactMatch